Extension/lib/utils/element-collapser.js

```diff
@@ -224,6 +226,17 @@ var ElementCollapser = (function() {
     };
 
     /**
+     * Prepares selector or rule text
+     *
+     * @param selector
+     * @param useShadowDom
+     * @returns {*}
+     */
+    var prepareSelector = function (selector, useShadowDom) {
```

Comment:
You should either pass `!!shadowRoot` as a parameter, or rename parameter to `shadowRoot` and use `return !!shadowRoot ? "::content " + selector : selector`